scrapers/AlexLegend/AlexLegend.yml

```diff
@@ -0,0 +1,48 @@
+name: AlexLegend
+
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - alexlegend.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[contains(concat(" ", normalize-space(@class), " "), " content-title ")]/h1
+      Code:
+        selector: //link[@rel="canonical"]/@href
+        postProcess:
+          - replace:
+            - regex: .+-(\d+)\..+
+              with: $1
+      Details: //div[contains(concat(" ", normalize-space(@class), " "), " content-desc ")][2]
+      Date:
+        selector: //div[contains(concat(" ", normalize-space(@class), " "), " content-date ")]/div[contains(concat(" ", normalize-space(@class), " "), " label ")]
+        postProcess:
+          - parseDate: 02.01.2006
+      Image:
+        selector: //script[contains(text(),"myFluidPlayer")]
+        postProcess:
+          - replace:
+            - regex: .+posterImage\s?:\s?'(.*?)'.+

```

Comment:
`posterImage : '(.+?)'` since you're escaping a single space

Reply:
This regex is run against a block of code which is likely to become changed in the future (and is likely dynamically generated). The group should be (.+?) instead of (.*?) but the spaces should be handled as something like a site library update would otherwise break the expression or match unrelated sections of the script.

Reply:
updated to not match empty image value